spec/presenters/email_alert_presenter_spec.rb

```diff
@@ -21,6 +22,21 @@
         expect(presented_data[:tags][:case_type]).to eq(product_safety_alert_payload["details"]["metadata"]["case_type"])
         expect(presented_data[:links]).to eq({})
       end
+
+      context "drcf_digital_markets_research finder" do
+        before do
+          stub_publishing_api_has_item(drcf_digital_markets_research_payload)
+        end
+
+        it "does not send the links hash in the payload for email alert api" do
+          drcf_digital_markets_research = DrcfDigitalMarketsResearch.find(drcf_digital_markets_research_payload["content_id"], drcf_digital_markets_research_payload["locale"])
+          email_alert_presenter = described_class.new(drcf_digital_markets_research)
+          presented_data = email_alert_presenter.to_json
+          expect(presented_data[:tags][:format]).to eq(drcf_digital_markets_research_payload["document_type"])
```

Comment:
Minor stylistic nit pick thing, but a space between setup and the expectations makes it slightly nicer to read.

Reply:
amended